### PR TITLE
Clean up jekyll config and add dependabot.

### PR DESCRIPTION
**Pull request by MarkLodato**
- Update environment to match GitHub Pages exactly.
- Update the instructions to fully explain how to test locally.
- Remove the .DS_Store file, which should not have been checked in.
- Exclude some files from Jekyll that shouldn't be published to the web.
- Add dependabot to update the Gemfile.lock.

### CODE REVIEW

**Review by joshuagl — COMMENTED** [review]
Thanks for this, I've made one suggestion where I felt extra detail would help, one to add a gem which I needed for the instructions to work here, and one to remove a duplicated package in the `apt install` command.

File: docs/README.md [hunk 1]
```diff
@@ -1,16 +1,52 @@
...
+3.  Install the dependencies locally:
+
+    ```bash
+    bundle config set --local path 'vendor/bundle'
+    bundle install
+    ```
+
+4.  (optional) Create a GitHub
+    [personal access token](https://github.com/settings/tokens/new) and add it
+    to your `~/.netrc`, like so:
```
[the remark this answers is not in the document]
Comment (MarkLodato) [reply docs/README.md:33]: Good point! Done.

**Comment by MarkLodato** [comment]
Done. Thanks for the suggestions!

**Review by joshuagl — APPROVED** [review]
LGTM, thanks.

**Pull request merged**